pkgs/desktops/gnome-3/extensions/workspace-matrix/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, findutils, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-workspace-matrix";
+  version = "3.0.0";
+
+  src = fetchFromGitHub {
+    owner = "mzur";
+    repo = "gnome-shell-wsmatrix";
+    rev = "v${version}";
+    sha256 = "1fgyzmd16kklcca7600bwg8w8pbb4klmapqsvmahlwa99vmkhfkn";
+  };
+
+  uuid = "wsmatrix@martin.zurowietz.de";
+
+  buildInputs = [
+    findutils
```

Comment:
`findutils` is used for the `schemas` target in the makefile.
So this should be in `nativeBuildInputs`.

Reply:
Ah, yes.  `glib` is similarly used as a command invoked during build rather than linked against, so I've made them both `nativeBuildInputs`.  Thanks!